### PR TITLE
PCM-1473: add revert message to Form Modal button messages

### PR DESCRIPTION
**Pull request by vidurar**
add revert message to Form Modal button messages for use in MC-frontend in context of custom form controls, e.g.

`label={this.props.intl.formatMessage(FormModalPage.Intl.revert)}`

as a part of this ticket https://jira.commercetools.com/browse/PCM-1473

### CODE REVIEW

**Comment by tdeekens** [comment]
Please build translations just as in the MC: https://github.com/commercetools/merchant-center-application-kit/blob/master/package.json#L9

File: packages/i18n/data/core.json [hunk 1]
```diff
@@ -47,6 +47,7 @@
   "Components.Dialog.labelConfirm": "Confirm",
   "Components.Dialog.labelCreate": "Create",
   "Components.Dialog.labelDelete": "Delete",
+  "Components.Dialog.labelRevert": "Revert",
```
Comment (vidurar) [inline packages/i18n/data/core.json]: @tdeekens translations built

**Comment by tdeekens** [comment]
> The button-messages.js will be removed soon.

Do we have that written down somewhere? In the file, issue or docs? It's a bit hard to contribute if not :)

**Comment by emmenko** [comment]
It's not documented true, I meant do that this week and didn't think someone would need to deal with those messages so soon 😅

**Comment by emmenko** [comment]
@vidurar let me open a PR to clean up the messages first, as we shouldn't define them in the app-components package anymore.

**Comment by emmenko** [comment]
PR is up: #1130

Once merged, you can update your PR with only the necessary changes.

**Comment by emmenko** [comment]
@vidurar btw, sorry for the trouble and thanks for contributing! 🙏

**Pull request merged**